includes/apple-exporter/builders/class-text-styles.php

```diff
@@ -89,69 +92,18 @@ public function register_style( $name, $values ) {
 	 * @access private
 	 */
 	private function add_html_styles() {
-
-		// Get information about the currently loaded theme.
-		$theme = \Apple_Exporter\Theme::get_used();
-
-		$conditional = [];
-		if ( ! empty( $theme->get_value( 'monospaced_color_dark' ) ) ) {
-			$conditional = [
-				'conditional' => [
-					'textColor'  => $theme->get_value( 'monospaced_color_dark' ),
-					'conditions' => [
-						'minSpecVersion'       => '1.14',
-						'preferredColorScheme' => 'dark',
-					],
-				],
-			];
+		// Try to get the text styles spec from the body component.
+		$body        = new Body();
+		$specs       = $body->get_specs();
+		$text_styles = $specs['default-text-styles'];
+		if ( empty( $text_styles ) || ! $text_styles instanceof Component_Spec ) {
```

Comment:
- The `empty` check is unnecessary given the `instanceof` check.
- An `empty` check could be used instead prior to accessing `$specs['default-text-styles']`, just to be sure.

Reply:
Good call. I'll update.